actgen/agents/action_dqn.py

```diff
@@ -8,8 +8,8 @@ class ActionDQNAgent(DQNAgent):
     def __init__(self, observation_space, action_space, params):
         super().__init__(observation_space, action_space, params)
 
-    def save(self, is_best, seed):
-        self.q.save('action_qnet' + f'_seed{seed}', 'results/', is_best)
```

Comment:
This is starting to get a bit unwieldy. I think I'd prefer to have the caller figure out what to name the network, and then just pass the filename in directly.